packages/preview/tud-letter/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+entrypoint = "src/lib.typ"
+name = "tud-letter"
+version = "0.1.0"
+compiler = "0.13.0"
+authors = ["Hugo Ledoux <@hugoledoux>"]
+repository = "https://github.com/hugoledoux/tudletter"
+description = "A Typst template that (almost?) replicates the TUDelft letter"
```

Comment:
The word typst is somewhat redundant here and should be omitted. See: https://github.com/typst/packages/blob/main/docs/manifest.md#writing-a-good-description